app/assets/stylesheets/partials/_layout.scss

```diff
@@ -1,4 +1,4 @@
 
-.back-to-top-panel {
-  margin-top: 5em
-}
\ No newline at end of file
+.mt-5 {
+  margin-top: 5rem;
```

Comment:
`rem`? Should it be `em`?

Reply:
`em` sets the size relatively to the container it is used in. `rem` is relative to a base size. So  it is more predictable to use the latter.